ledger/internal/applications.go

```diff
@@ -44,7 +44,7 @@ type cowForLogicLedger interface {
 	SetKey(addr basics.Address, aidx basics.AppIndex, global bool, key string, value basics.TealValue, accountIdx uint64) error
 	DelKey(addr basics.Address, aidx basics.AppIndex, global bool, key string, accountIdx uint64) error
 
-	round() basics.Round
+	Round() basics.Round
```

Comment:
looks like it should not be exported

Reply:
Thanks, I agree, that was left over from another path.  I'll revert.